_drafts/2020-04-15-visual-regression-testing.md

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
remove this file, it's not a draft anymore and it's creating 2 articles in the preview (weird behavior btw)